index.html

```diff
@@ -56,5 +56,191 @@ <h1 class="Intro">
         </li>
       </ul>
     </section>
+
+    <section class="work--section">
+      <div class="parent--container">
+        <div class="first--child">
+          <a href="#"><img src="images/2nd pic.svg" alt="" /></a>
+        </div>
+
+        <div class="second--child">
+          <h1>Tonic</h1>
+          <ul class="ul--list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">
+            <a class="see--project--link" href="#">See project</a>
+          </button>
```

Comment:
- [ ] To ensure consistency with the Figma design, it is crucial to align the button interaction style accordingly. 🥱🥱
- [ ] It appears that your current button style does not match the Figma design. As per the project requirements, it is essential to strictly adhere to the specified button interaction style from the Figma design. 
- [ ] Please make the necessary adjustments to ensure that your button style strictly follows the Figma design, maintaining visual coherence and meeting the project requirements.
- [ ] Figma 
![Screenshot 2023-06-03 200911](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/118761303/7cf9fb4c-23a7-42df-ba6b-0deebbf5db62)

- [ ] Without pressing/hovering your button style:
![Screenshot 2023-06-03 201521](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/118761303/e6ba4979-e8d7-48ab-8bd8-1e69b02086af)